### PR TITLE
Add v2.2.0 changelog

### PR DESCRIPTION
**Pull request by davidz627**
Changelog for v2.2.0

/assign @msau42 @lpabon 
/cc @jsafrane 

/kind documentation

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG-2.1.md [hunk 1]
```diff
@@ -1,16 +0,0 @@
-# Changelog since v2.0.1
```
Comment (msau42) [inline CHANGELOG-2.1.md]: we should keep this file?

Comment (davidz627) [reply CHANGELOG-2.1.md]: oh I only saw 1 file so I assumed we were doing replacements

Comment (davidz627) [reply CHANGELOG-2.1.md]: fixed

**Pull request merged**